### PR TITLE
add more context to dialog title shown after sendToChat()

### LINKED ISSUE #2594: add more context to dialog title shown after `sendToChat()`
the title should be set to `send_file_to` or `send_message_to`. in case the RelayHelper is not used from `sendToChat()`, the title should stay as is.

for more context and screenshots, see https://github.com/deltachat/deltachat-ios/pull/1907 and https://github.com/deltachat/deltachat-android/pull/2592

### PR DESCRIPTION
**Pull request by adbenitez**
close #2594

### CODE REVIEW

**Review by r10s — APPROVED** [review]
the the getter/setter use the wording "shared title" while the extra is called "sharing title" - this is unnecessarily confusing, better use the same wording; so eg. use `SHARED_TITLE` as the name for the extra.

otherwise lgtm.

File: src/org/thoughtcrime/securesms/util/RelayUtil.java [hunk 1]
```diff
@@ -79,6 +80,14 @@ public static String getSharedText(Activity activity) {
         }
     }
 
+    public static String getSharedTitle(Activity activity) {
+        try {
+            return activity.getIntent().getStringExtra(SHARING_TITLE);
```
Comment (adbenitez) [inline src/org/thoughtcrime/securesms/util/RelayUtil.java:85]:
```suggestion
            return activity.getIntent().getStringExtra(SHARED_TITLE);
```

File: src/org/thoughtcrime/securesms/util/RelayUtil.java [hunk 2]
```diff
@@ -123,6 +132,10 @@ public static void setSharedText(Intent composeIntent, String text) {
         composeIntent.putExtra(IS_SHARING, true);
     }
 
+    public static void setSharedTitle(Intent composeIntent, String text) {
+        composeIntent.putExtra(SHARING_TITLE, text);
```
Comment (adbenitez) [inline src/org/thoughtcrime/securesms/util/RelayUtil.java:136]:
```suggestion
        composeIntent.putExtra(SHARED_TITLE, text);
```

**Comment by adbenitez** [comment]
> the the getter/setter use the wording "shared title" while the extra is called "sharing title" - this is unnecessarily confusing, better use the same wording; so eg. use `SHARED_TITLE` as the name for the extra.

good point, fixed!

**Pull request merged**